plyer/platforms/android/smsreceive.py

```diff
@@ -0,0 +1,55 @@
+'''
+Android Recieve SMS
+-----------
+'''
+
+from jnius import autoclass
+from plyer.facades import SmsReceive
+from jnius import PythonJavaClass
+from jnius import java_method
+
+
+Intent = autoclass('android.content.Intent')
+Bundle = autoclass('android.os.Bundle')
+GenericBroadcastReceiver = autoclass(
+    'org.renpy.android.GenericBroadcastReceiver'
+)
+SmsMessage = autoclass('android.telephony.SmsMessage')
+Toast = autoclass('android.widget.Toast')
+
+
+class BroadcastReceiver(PythonJavaClass):
+    __javainterfaces__ = [
+        'org/renpy/android/GenericBroadcastReceiverCallback']
```

Comment:
the new one is [`org.kivy.android`](https://github.com/kivy/python-for-android/blob/master/pythonforandroid/bootstraps/sdl2/build/src/org/kivy/android/GenericBroadcastReceiver.java)